dev/com.ibm.ws.jca.cm/resources/com/ibm/ws/jca/cm/internal/resources/J2CAMessages.nlsprops

```diff
@@ -73,3 +73,7 @@ INVALID_MBEAN_INVOKE_ACTION_J2CA8061.useraction=Change the action being invoked
 MBEAN_ATTRIBUTE_NOT_FOUND_J2CA8062=J2CA8062E: The {0} attribute is not valid for the {1} MBean.
 MBEAN_ATTRIBUTE_NOT_FOUND_J2CA8062.explanation=An attempt was made to get or set an attribute that was not available for the specified MBean.
 MBEAN_ATTRIBUTE_NOT_FOUND_J2CA8062.useraction=Change the attribute specified to one of the available attributes for this MBean.
+
+CONNECTION_LEAK_DETECTED_J2CA8070=J2CA8070I: A connection that was obtained from {0} was not closed. The following stack identifies the code that obtained the connection: {1}
+CONNECTION_LEAK_DETECTED_J2CA8070.explanation=A connection was not closed by the code that obtained it. The stack identifies one area of the code where this is occurring, but there might also be others. In order to avoid cluttering the logs and minimize the impact of collecting stacks, only one occurrence is reported per server start.
```

Comment:
suggested edits:

`The stack identifies one area of the code where this is occurring, but there might also be others.`
--->
`The stack identifies one area of the code where an unclosed connection occurs, but other instances might exist elsewhere.`
 

`In order to avoid cluttering the logs and minimize the impact of collecting stacks, only one occurrence is reported per server start.`
--->
`To avoid cluttering the logs and minimize the impact of collecting stacks, only one occurrence is reported per server start.`